src/2048-expert/index.html

```diff
@@ -13,6 +13,22 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+        <ul>
+            <li>Take actions with arrow keys</li>
+            <li>Start a new game by pressing enter</li>
+            <li> Tiles with the same value will get merged after moves</li>
```

Comment:
To make it more clear why this game is called "2048 Expert", there could be additional information in the help dialogue such as, "After each move **two** new tiles will appear on the board"